docs/examples/guides/backends_guide.py

```diff
@@ -72,49 +72,58 @@
 
 
 # TrackingBackend creation example
-from orquestra.quantum.symbolic_simulator import SymbolicSimulator
-from orquestra.quantum.trackers import MeasurementTrackingBackend
+from orquestra.quantum.backends import MeasurementTrackingBackend, SymbolicSimulator
 
 backend = MeasurementTrackingBackend(SymbolicSimulator(), "tracker_example")
 # End TrackingBackend creation example
 
 
-# Importng and Exporting with different frameworks
-from orquestra.integrations.forest.conversions import (
-    export_to_pyquil,
-    import_from_pyquil,
-)
 from orquestra.integrations.cirq.conversions._circuit_conversions import (
     export_to_cirq,
     import_from_cirq,
 )
-from orquestra.integrations.qulacs.conversions import convert_to_qulacs
+
+# Importing and Exporting with different frameworks
+from orquestra.integrations.forest.conversions import (
+    export_to_pyquil,
+    import_from_pyquil,
+)
 from orquestra.integrations.qiskit.conversions import (
     export_to_qiskit,
     import_from_qiskit,
 )
+from orquestra.integrations.qulacs.conversions import convert_to_qulacs
 
 # end importing/exporting examples
-
-
 # Inherit QuantumSimulator
-from orquestra.quantum.api.backend import QuantumSimulator
+from orquestra.quantum.api.backend import QuantumBackend
+from orquestra.quantum.measurements import Measurements
 
 
-class mysimulator(QuantumSimulator):
+class MyBackend(QuantumBackend):
```

Comment:
So the example fits the docs. The docs talk about "making your own backend" and then the example is a simulator. I thought this would be less confusing.

Reply:
Ok, we changed to runners, so when you merge in changes from `main` it will be irrelevant anyway 🤷‍♂️